bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -585,5 +588,187 @@ contract("Bridge NFT", async function(accounts) {
         );
       });
     });
+
+    describe("acceptTransfer", async function() {
+      let symbol;
+      let name;
+      let baseURI;
+      let contractURI;
+      let tokenAddress;
+      const tokenId = 1;
+      const blockHash = randomHex(32);
+      const transactionHash = randomHex(32);
+      const logIndex = 1;
+
+      beforeEach(async function () {
+        symbol = await this.token.symbol();
+        name = await this.token.name();
+        baseURI = await this.token.baseURI();
+        contractURI = await this.token.contractURI();
+        tokenAddress = this.token.address;
+        let receipt = await executeSideNFTTokenCreationTransaction.call(
+            this,

```

Comment:
this would be the javascript test object, is this ok?

Reply:
yeah this was some automatic refactor stuff - I'll remove it 😅